src/osd/ECBackend.cc

```diff
@@ -2414,6 +2415,7 @@ void ECBackend::be_deep_scrub(
   const hobject_t &poid,
   uint32_t seed,
   ScrubMap::object &o,
+  std::atomic<bool> *preempted,
```

Comment:
It doesn’t look like they’re ever set internally, so why are these pointers instead of const references?